tests/util/jsx.js

```diff
@@ -71,11 +69,27 @@ describe('jsxUtil', () => {
     });
 
     it('Can ignore null', () => {
-      assertInValid(`
+      assertValid(`
```

Comment:
Does this mean that `Test` is changing from "not a component" to "a component", or the reverse?

Reply:
`assertInValid` wasn't doing anything - `isReturningJSX` was being double-negated on L31 - it was identical in function to `assertValid`

Reply:
gotcha